docs/content/develop/make-a-breaking-change.md

```diff
@@ -181,6 +181,21 @@ Example (`google_container_cluster.enable_shielded_nodes`)
 * https://github.com/GoogleCloudPlatform/magic-modules/pull/5263 changed the default value for the field and added an upgrade guide entry
 * Note: In 4.0.0 the upgrade guide was split between main and the major release branch. Going forward, those changes will be made against main exclusively.
 
+## Contributing to the next major release (`5.0.0`)
```

Comment:
I'd suggest making this a subheading of `## Making a breaking change` (i.e. three `#`s) before `Renaming a Field`